executor/src/witgen/processor.rs

```diff
@@ -24,6 +24,27 @@ use super::{
 
 type Left<'a, T> = Vec<AffineExpression<AlgebraicVariable<'a>, T>>;
 
+/// The data mutated by the processor
```

Comment:
Is this distinctive enough from the MutableState?

Reply:
Renamed it to `SolverState`, what do you think?